packages/web-app-admin-settings/src/components/Users/CreateUserModal.vue

```diff
@@ -17,6 +17,7 @@
           :label="$gettext('User name') + '*'"
           :error-message="formData.userName.errorMessage"
           :fix-message-line="true"
+          autocomplete="off"
```

Comment:
autocomplete="off" doesn't help unfortunately. See https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion#preventing_autofilling_with_autocompletenew-password

Reply:
but isn't it supposed to be on passwords only 🤔

Reply:
"If you are defining a user management page where a user can specify a new password for another person, and therefore you want to prevent autofilling of password fields, you can use autocomplete="new-password""

Reply:
will research how to do it